adhocracy-plus/assets/scss/components/_menu_layout.scss

```diff
@@ -1,27 +1,27 @@
-.l-menu__menu,
-.l-menu__content,
-.l-menu__aside {
-    margin-bottom: 2em;
-}
+// .col-md-3,
+// // .l-menu__content,
+// .col-md-3 {
+//     margin-bottom: 2em;
+// }
 
 // .l-menu {
 //     display: flex;
 //     flex-wrap: wrap;
 //     // @extend .row;
 // }
 
-.l-menu__menu {
-    @extend .col-md-3;
-}
+// .col-md-3 {
+//     @extend .col-md-3;
+// }
 
-.l-menu__content {
-    @extend .col-md-9;
-}
+// .l-menu__content {
+//     @extend .col-md-9;
+// }
 
-.l-menu__content--aside {
-    @extend .col-md-6;
-}
+// .l-menu__content--aside {
+//     @extend .col-md-6;
+// }
 
-.l-menu__aside {
-    @extend .col-md-3;
-}
+// .col-md-3 {
```

Comment:
commented out, but aren't you using it below in `base_dashboard_project.html`?

Reply:
thats a bootstrap default class, it'll get deleted in next stage after i've checked mobile, also had a look on dev as my database was empty so had to make a few projects but need to do more testing